src/rust/grapl-web-ui/src/authn/user.rs

```diff
@@ -24,11 +24,11 @@ pub struct AuthenticatedUser {
 
 #[derive(thiserror::Error, Debug)]
 pub enum UserAuthenticationError {
-    #[error("unable to validate user session")]
+    #[error("invalid user session")]
     SessionValidation(#[source] AuthDynamoClientError),
-    #[error("unable to get user data from validated session")]
+    #[error("session validated but user `{}` not found in user table 🤔", .0)]
```

Comment:
Do we want to embed emojis in our logs? 🤔

Reply:
Yeah, I wasn't sure but I asked myself "why not? why so serious?" 🤷 

I'm fine with removing it in case anyone thinks we should.

Reply:
I love emojis as much as the next guy, but I'd rather be conservative with logs and stick to boring ASCII characters. I imagine most log processing software should be able to cope, but I wouldn't want to tempt fate.

Reply:
Yeah, I'll remove.

Reply:
Emojis can show up for other reasons, but I think your point remains.